CHANGELOG.md

```diff
@@ -172,6 +172,10 @@
   [JP Simard](https://github.com/jpsim)
   [#4985](https://github.com/realm/SwiftLint/issues/4985)
 
+* Do not trigger `prefer_self_in_static_references` rule on class declarations.  
+  [kasrababaei](https://github.com/kasrababaei)
+  [#5084](https://github.com/realm/SwiftLint/issues/5084)
```

Comment:
This line is supposed to refer to an issue given that it exist. For your finding, there isn't an issue so the line can be omitted.